### PR TITLE
[[ Bug 22412 ]] Save tabAlign when tabStops is not set

### PR DESCRIPTION
**Pull request by montegoulding**
This patch ensures that if the `tabAlign` property of a field is set but the
`tabStops` is not then it will still be saved.

### CODE REVIEW

File: tests/lcs/core/field/softkeyboardproperties.livecodescript [hunk 1]
```diff
@@ -17,10 +17,16 @@ You should have received a copy of the GNU General Public License
...
+	TestPropRoundTrip "field", "keyboardType", tValues
 end TestKeyboardTypeRoundTrip
 
 on TestReturnKeyTypeRoundTrip
-	TestEnumPropRoundTrip "field", "returnKeyType", ",default,go,google,join,next,route,search,send,yahoo,done,emergency call"
+	local tValues
+	put ",default,go,google,join,next,route,search,send,yahoo,done,emergency call" into tValues
+	split tValues by comma
+	TestPropRoundTrip "field", "returnKeyType", tValues
 end TestReturnKeyTypeRoundTrip
```
Comment (livecodeian) [inline tests/lcs/core/field/softkeyboardproperties.livecodescript]: I don't think these tests are intended for this PR

**Comment by livecodeian** [comment]
@livecode-vulcan review ok 5a16185

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeian ok 5a161856936c829675313ef7bf297da406f2318c

**Comment by livecode-vulcan** [comment]
😎 test success 5a161856936c829675313ef7bf297da406f2318c

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/693)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/102)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/102)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/102)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/102)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2219)
* try-community-universal-ios-iphoneos13.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.1/builds/53)
* try-community-universal-ios-iphonesimulator13.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.1/builds/53)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2248)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2235)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2234)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2268)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2208)

**Pull request merged**